<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame_4</class>
 <widget class="QFrame" name="Frame_4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="unit4D">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-pcd_project1-Desktop_Qt_5_8_0_GCC_64bit-Debug/unitd.jpg</normaloff>../build-pcd_project1-Desktop_Qt_5_8_0_GCC_64bit-Debug/unitd.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="unit4C">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-pcd_project1-Desktop_Qt_5_8_0_GCC_64bit-Debug/unitc.jpg</normaloff>../build-pcd_project1-Desktop_Qt_5_8_0_GCC_64bit-Debug/unitc.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="unit4B">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-pcd_project1-Desktop_Qt_5_8_0_GCC_64bit-Debug/unitb.jpg</normaloff>../build-pcd_project1-Desktop_Qt_5_8_0_GCC_64bit-Debug/unitb.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="unit4A">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-pcd_project1-Desktop_Qt_5_8_0_GCC_64bit-Debug/unita.jpg</normaloff>../build-pcd_project1-Desktop_Qt_5_8_0_GCC_64bit-Debug/unita.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
